src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+import java.util.Scanner;
+import java.util.Arrays;
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static int count = 0;
+    public static int seq = 1;
+    public static Task[] list = new Task[100];
+
+    public static void sort(String t) throws DukeException { //classify tasks between todo / deadline / event
```

Comment:
You may consider replacing method names (such as sort and echo in your case) to explain the named entity to the reader accurately and with sufficient level of detail.